src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/allocations/rules/FemaleAssessmentRule.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.allocations.rules
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.allocations.UserAllocatorRule
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.allocations.UserAllocatorRuleOutcome
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ApprovedPremisesApplicationEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.AssessmentEntity
+
+@Component
+@ConditionalOnProperty(name = ["user-allocations.rules.female-assessments.enabled"])
+class FemaleAssessmentRule(
+  @Value("\${user-allocations.rules.female-assessments.priority:0}")
+  override val priority: Int,
+  @Value("\${user-allocations.rules.female-assessments.allocate-to-user}")
+  private val allocateToUser: String,
```

Comment:
I think as with the ESAP rule, we're more than likely going to have multiple people qualified to carry out Female assessments, so I'd feel happier if we used `AllocateByQualification` here

Reply:
I've updated this to use `AllocateByQualification(UserQualification.WOMENS)` instead of a specific user.